src/site/components/events_list.drupal.liquid

```diff
@@ -1,6 +1,9 @@
 <div class="events-listing events-show">
+  {% assign count = 0 %}
   {% for event in fieldOffice.entity.reverseFieldOfficeNode.entities %}
   {% if event.title.length%}
+  {% assign count = count | plus: 1 %}
+  {% if count != 0 %}
```

Comment:
Can `count == 0` here?

Reply:
Yep. If we get an event object without a title in our array (and it does happen for various reasons), and nothing else, we are going to have an event, but since there is no title, we are in the loop with a 0 count. Better safe than sorry kind of thing, @cvalarida

Reply:
Right, that makes sense, but we're nesting this `if count != 0` under the assignment where we're definitely adding 1, right?

```javascript
let count = 0
for (event in entities) {
  if (event.title.length) {
    count++;
    if (count != 0) { ... } // When it gets here, it'll always be at least 1, right?
  }
}
```

Reply:
You're right, @cvalarida - taking that conditional out.